analyzer/src/funTest/kotlin/managers/GoModFunTest.kt

```diff
@@ -95,10 +96,9 @@ class GoModFunTest : StringSpec({
     "Local module dependencies make the analysis fail" {
         // TODO: Implement support for local dependencies, see https://github.com/oss-review-toolkit/ort/issues/7649.
         val definitionFile = testDir.resolve("gomod-submodules/app/go.mod")
-        val expectedResultFile = testDir.resolve("gomod-submodules-embed-expected-output.yml")
 
         val result = create("GoMod").resolveSingleProject(definitionFile)
 
-        result.withInvariantIssues().toYaml() should matchExpectedResult(expectedResultFile, definitionFile)
+        result.issues.size shouldBe 1
```

Comment:
Nit (sorry for not realizing earlier): Could also be `shouldHaveSize 1`.

Reply:
I realize we have more such code that could be changed, so I'll probably bulk-change after the merge of this.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/7716.